docs/recipes/Deploying-Modules.md

```diff
@@ -0,0 +1,156 @@
+      run: npm run build
+      env:
+        NODE_ENV: production
+    - name: 'Deploy'
+      run: now build --prod --confirm -t $NOW_TOKEN
+      env:
+        NOW_TOKEN: ${{ secrets.NOW_TOKEN }}
+```
+
+If you view the action running, during the `Deploy` step, you should see something like
+
+```bash
+Run now build --prod --confirm -t $NOW_TOKEN
+Now CLI 17.1.1
+- Setting up project
+Linked to [YOUR NOW USERNAME]/[YOUR-REPO-NAME] (created .now and added it to .gitignore)
+Inspect: https://zeit.co/[YOUR NOW USERNAME]/[YOUR-REPO-NAME]/
+- Building
+- Completing
+Production: https://[YOUR-REPO-NAME].now.sh
+https://[YOUR-REPO-NAME]-[HASH].now.sh
+```
+
+## Create and Update Your Module Map
+
+You will now need to add the module you just deployed to your module map.
+
+### Creating a Module Map in Now.sh
+
+To create a module map in Now, you can create a module-map.json locally that looks something like this:
+
+```json
+{
+  "modules": {
+
+  }
+}
+```
+
+You can deploy this by running the [now](https://zeit.co/docs/now-cli#commands/now/basic-usage) command within the directory that you created the `module-map.json` in. You can now view it at the production URL that `now` prints out, you will need that URL for the following steps.
+
+### Create an Update Module Script
+
+```javascript
+const fs = require('fs-extra');
+const fetch = require('node-fetch');
+const { name, version } = require('../package.json');
+
+// This is created during the build process within the deploy action
+const bundleIntegrity = require('../bundle.integrity.manifest.json');
+
+const moduleMapUrl = 'YOUR MODULE MAP URL'; // This is the module map URL you got in the previous step
+
+const STATIC_ASSETS_URL = 'URL TO WHERE YOUR MODULE IS DEPLOYED'; // example 'https://my-module.now.sh'
+
+const updateModuleMap = async () => {
+  try {
+    const response = await fetch(moduleMapUrl);
+
+    const moduleMapContent = await response.json();
+    const dir = 'tmp';
+
+    moduleMapContent.modules[name] = {
+      browser: {
+        url: `${STATIC_ASSETS_URL}/${version}/${name}.browser.js`,
+        integrity: bundleIntegrity.browser,
+      },
+      legacyBrowser: {
+        url: `${STATIC_ASSETS_URL}/${version}/${name}.legacy.browser.js`,
+        integrity: bundleIntegrity.legacyBrowser,
+      },
+      node: {
+        url: `${STATIC_ASSETS_URL}/${version}/${name}.node.js`,
+        integrity: bundleIntegrity.node,
+      },
+    };
+
+    await fs.ensureDir(dir);
+    await fs.writeFile(
+      './tmp/module-map.json', JSON.stringify(moduleMapContent, null, 2)
+    );
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+updateModuleMap().catch((err) => {
+  // eslint-disable-next-line no-console
+  console.log(err);
+});
+```
+
+### Update your Deploy Action to Update your Module Map
+
+At the bottom of your deploy action, you will want to add the following:
+
+```yml
+ - name: 'Update Module Map'
+      run: |
+        node ./scripts/updateModuleMap.js
```

Comment:
Maybe be explicit in where to put the `updateModuleMap.js` script inside `### Create an Update Module Script`

Reply:
We need to have a broader discussion about where this script will live, it needs to be 100% reusable by all modules/static host providers and it should be either shipped with generator or added to the CLI

Reply:
We will need to create tickets for evolving this Guide as this will become a multi-topic/multi-provider guide on running this. For now, starting small and simple is good but we will need to go places with this in the future.

Reply:
Thanks for updating the description for `### Create an Update Module Script`